site/public/js/server.js

```diff
@@ -1783,7 +1785,7 @@ function dynamicScrollContentOnDemand(jElement, urlPattern, currentThreadId, cur
     dynamicScrollLoadIfScrollVisible(jElement);
     $(jElement).scroll(function(){
         var element = $(this)[0];
-        var sensitivity = 3;
+        var sensitivity = 1.5;
```

Comment:
From our discussion, this was modified to fix the vertical cutoff of the thread list? On chrome it is still being cutoff for me.

Reply:
Seems like this is happening on master?? Not sure when this began...